2.0/trigger_daily_flow.py

```
@@ -0,0 +1,26 @@
+import asyncio
+import logging
+from datetime import datetime, timedelta
+from prefect.client import get_client
+import config.settings as settings
```

Pylint (3.3.7)

[refactor] Use 'from config import settings' instead

(R0402)